docs/library/cohesity_restore_vm.md

```diff
@@ -45,19 +45,24 @@ This Ansible Module supports Physical and GenericNas environments and initiates
     endpoint: <identifies the source endpoint to which the the restore operation will be performed>
     backup_id: <optional Cohesity Backup Run ID for the restore operation.  If not selected, the most recent RunId will be used>
     backup_timestamp: <not implemented>
-    vm_names:
```

Comment:
This will break backward compatibility ?

Reply:
Existing vm_names doesn't work, Shall we add fixto check for both vms and vm_names?

Reply:
Doesn't work == with new params or didnt work from start , if its the latter, lets change it. else preserve backward compatibility

Reply:
When I tried with old params, the vm names were not passed successfully. On debugging found out that the tasks were expecting variable with name **`vms`** not **`vm_names`**.
In the sample playbooks provided, the variable name was incorrect(vm_names).
This will support both the variable names.
**_`"{{ cohesity_restore_vm.vms | default(cohesity_restore_vm.vm_names | default('')) }}"`_**